src/i18n/tr.yaml

```diff
@@ -0,0 +1,518 @@
+Are you sure you want to disconnect all websites?: Tüm web sitelerinin bağlantısını kesmek istediğinizden emin misiniz?
+$dapps-description: MyTonWallet'ı destekleyen Dapp'lerde oturum açabilirsiniz. Bu Dapp'ler, cüzdan adresini, bakiyeyi görüntüleyebilir ve işlem talep edebilirler.
+Logged in with MyTonWallet: MyTonWallet ile oturum açıldı
+No active connections: Aktif bağlantı yok
+Unstaked successfully: Başarılı şekilde unstake edildi
+Other: Diğer
+Assets & Activity: Varlıklar ve Etkinlik
+Tokens Settings: Token ayarları
+Your Tokens: Token'larınız
+Add Token: Token ekle
+Delete Token: Token sil
+Are you sure you want to delete?: "%token%'ı silmek istediğinizden emin misiniz?"
+Hide Tokens With No Balance: Sıfır bakiyeli tokenları gizle
+Hide Tokens With No Price: Fiyatı olmayan tokenları gizle
+Sort By Value: Değere göre sırala
+Update: Güncelle
+Restart: Yeniden başlat
+Update MyTonWallet: MyTonWallet'ı güncelle
+Connect Ledger: Ledger'i bağla
+Connecting to your Ledger...: Ledger'ınıza bağlanıyor...
+Failed to connect: Bağlantı başarısız
+Ledger Connected!: Ledger bağlandı!
+Select Ledger Wallets: Ledger cüzdanları seçin
+"%1$d Selected": "%1$d seçildi"
+Add: Ekle
+Try Again: Yeniden dene
+Driver: Sürücü
+Connect your Ledger to PC: Ledger'ınızı PC'ye bağlayın
+Unlock it and open the TON App: Cihazın kilidi açın ve TON uygulamasını başlatın
+Maybe you need to check the password or change the driver.: Şifreyi kontrol etmeniz veya sürücüyü değiştirmeniz gerekebilir.
+Or import from...: Ya da içeri aktarmanız gerekebilir...
+Confirm via Ledger: Ledger aracılığıyla onaylayın
+Please confirm transaction on your Ledger: Lütfen Ledger'ınızdaki işlemi onaylayın
+Declined: Reddedildi
+Create Wallet: Cüzdan Oluşturun
+Switch to the newly opened tab to connect Ledger.: Ledger'ı bağlamak için yeni açılan sekmeye geçin.
+Once connected, switch back to this window to proceed.: Bağlandıktan sonra devam etmek için bu pencereye geri dönün.
+Not all transactions were sent successfully: Tüm işlemler başarılı şekilde gönderilmedi
+The time on your device is incorrect, sync it and try again: Cihazınızdaki saat hatalı, saati senkronize edin ve tekrar deneyin
+Name or Address...: İsim veya Adres...
+Such error, many tabs: Bu tür hatalar, birçok sekme
+$many_tabs_error_description: |
+  MyTonWallet, uygulama ile yalnızca bir etkin sekmeyi destekler.
+  Bu sekmeyi kullanmaya devam etmek için lütfen sayfayı yeniden yükleyin ya da kapatın.
+Reload App: Uygulamayı yeniden yükleyin
+Please confirm operation on your Ledger: Lütfen Ledger'ınızdaki işlemi onaylayın
+Token Not Found: Token bulunamadı
+Try another keyword or address.: Başka bir anahtar kelime veya adres deneyin.
+Decrypt: Şifre çöz
+Encrypted Message: Şifrelenmiş Mesaj
+Message is encrypted.: Mesaj şifrelenmiştir.
+$dapp_ledger_warning1: **Ledger** cüzdanınızı kullanarak çok yönlü bir işlem göndermek üzeresiniz. İlgili her işlemi **tek tek manuel olarak** imzalamanız gerekecektir.
```

Comment:
```suggestion
$dapp_ledger_warning1: "**Ledger** cüzdanınızı kullanarak çok yönlü bir işlem göndermek üzeresiniz. İlgili her işlemi **tek tek manuel olarak** imzalamanız gerekecektir."
```

Reply:
I was not informed about the rule about [**... → `: "**..."].   (As a translator I have limited knowledge about coding spelling rules). Are you going to edit by yourself, or would you like me to handle the editing of the rest of the file?